README.md

```diff
@@ -176,6 +176,11 @@ when using the hardware engine. It defaults to *redis*, with no domain.
 ##### `LOCK_MANAGER_PORT`
 Port of the system where redis is running. Defaults to *6379*.
 
+##### `VANAGON_USE_MIRRORS`
+Controls whether component sources are downloaded directly from upstream URLs
+or from configured mirrors. Valid values are `y` or `n`. Set to `n` when
+building outside of the Puppet private network to skip unreachable mirrors.
```

Comment:
What's the default value?

Reply:
Dropped `y` and clarified that most Puppet Vanagon projects fetch from internal mirrors by default and that setting `VANAGON_USE_MIRRORS` to `n` will disable that behavior.